pac4j-core/src/main/java/org/pac4j/core/client/config/BaseClientConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package org.pac4j.core.client.config;
+
+import org.pac4j.core.util.InitializableObject;
+
+/**
+ * Base parent class for all client configurations
+ * to hold common fields or to be used as a common parent
+ * for type checking and casts in customizations.
+ *
+ * @author Misagh Moayyed
+ * @since 4.0.1
+ */
+public abstract class BaseClientConfiguration extends InitializableObject {
```

Comment:
I don't see any fields yet in this common class. Missing something?

Reply:
I didn't find a common fields for configurations, so I left it blank. My intention is to be able to use instanceof on configuration classes that are, per contract, expected to be of type `BaseClientConfiguration`, rather than a very generic `InitializableObject`. This is more of nice-to-have.

Reply:
I think the `logoutHandler` might fall into that category. Anyway for now. Merging.